src/main/kotlin/eu/kevin/api/models/exception/KevinApiClientErrorException.kt

```diff
@@ -0,0 +1,16 @@
+package eu.kevin.api.models.exception
+
+internal class KevinApiClientErrorException(
+    val response: ErrorResponse
+) : Exception()
+
+internal data class ErrorResponse(
+    val error: Error,
+    val data: Any
+)
+
+internal data class Error(
```

Comment:
I usually discourage naming classes under existing language class names, but probably in this case this will never be invoked outside of error serialization